cpp/arcticdb/version/version_core.cpp

```diff
@@ -1435,10 +1435,10 @@ VersionedItem collate_and_write(
     });
 }
 
-void delete_incomplete_keys(PipelineContext* pipeline_context, Store* store) {
+void delete_incomplete_keys(PipelineContext& pipeline_context, Store& store) {
```

Comment:
Guessing `PipelineContext` can be const&

Reply:
A rational person would think so, but because of irritating reasons the function returning the iterator into the pipeline context can't be made easily const, and we want this class to just go away so I don't want to spend too much time (any time) polishing it.